implementation/src/main/java/io/smallrye/jwt/KeyUtils.java

```diff
@@ -118,31 +138,51 @@ public static PublicKey decodeJWKSPublicKey(String jwksValue) throws GeneralSecu
             // A JWK
             jwk = jwks;
         }
-        String e = jwk.getString("e");
```

Comment:
@aaronanderson Thanks for taking on this legacy code as well. Instead of manually creating RS and EC keys here, please copy the code from `KeyLocationResolver` which reads JSON into JWK and returns a PublicKey